src/run/formlyNgModelAttrsManipulator.js

```diff
@@ -18,7 +18,10 @@ function addFormlyNgModelAttrsManipulator(formlyConfig, $interpolate) {
       return template;
     }
     el.innerHTML = template;
-    var modelNodes = el.querySelectorAll('[ng-model], [data-ng-model]');
+    var elementFilter = ':not([formly-skip-ng-model-attrs-manipulator])' +
+                        ':not(' + data.skipNgModelAttrsManipulator + ')';
```

Comment:
Awesome! I should have covered this in the tests, but what if I pass `false ` as the skipNgModelAttrsManipulator? :-)